src/app/components/client/toolbar/UpsellBadge.module.scss

```diff
@@ -55,6 +53,15 @@
   }
 }
 
+// Override the background of the base button component
+a.upsellBadge,
+button.upsellBadge {
+  /* stylelint-disable-next-line declaration-no-important */
+  background: none !important;
+  /* stylelint-disable-next-line declaration-no-important */
+  color: tokens.$color-black !important;
+}
+
```

Comment:
issue (blocking): This makes it risky to update the base button later, because whoever makes that edit likely won't know about these changes and whether they work well with it. (e.g. imagine a change that sets the border to transparent.) So I'd suggest either pulling these styles into that button component as another variant, or not using the base button.

`!important` also is too blunt an instrument to override styles, and should not be used. Instead, you could increase the specificity by e.g. adding the parent element to the selector, which would also emphasise that the change only applies in that context.